common/ustr.cpp

```diff
@@ -193,7 +181,7 @@ int U32String::vformat(const value_type *fmt, const value_type *fmtEnd, U32Strin
 			case 's':
 				string_temp = va_arg(args, char *);
 				tempPos = output.size();
-				output.insertString(string_temp, pos);
+				output.insertString(string_temp, pos, Common::kUtf8);
```

Comment:
That's really not a good thing to imply that %s is UTF-8 when formatting random stuff.
I wonder if this one should raise a warning or something.
Maybe use kASCII and issue warning?

Reply:
Since we have decided that escapes in strings are ok, I think it should be kUtf8